test/functional/interface_usdt_net.py

```diff
@@ -158,12 +156,15 @@ def handle_outbound(_, data, __):
         bpf.perf_buffer_poll(timeout=200)
 
         self.log.info(
-            "check that we got both an inbound and outbound version message")
+            "check receipt and content of in- and outbound version messages")
+        for event, inbound in events:
+            check_p2p_message(event, inbound)
```

Comment:
nit: clearer naming to indicate it's a bool
```suggestion
        for event, is_inbound in events:
            check_p2p_message(event, is_inbound)
```